kratos/tests/vtk_output_process_ref_files/ascii3D/Main_MovingNodes_0_6.vtk

```diff
@@ -28,11 +28,18 @@ VELOCITY 3 3  float
 0.00000000e+00 0.00000000e+00 4.00000000e+00 
 2.00000000e+00 0.00000000e+00 4.00000000e+00 
 2.00000000e+00 2.00000000e+00 4.00000000e+00 
+BOUNDARY 1 3  float
+-1.00000000e+00
+-1.00000000e+00
+-1.00000000e+00
 CELL_DATA 2
-FIELD FieldData 2
+FIELD FieldData 3
```

Comment:
why do those change?

Reply:
This changed because now there are three data on the cells DENSITY, YOUNGS_MOD and BOUNDARY

Reply:
Because the flag is one more variable

Reply:
ok thx